internal/monitors/collectd/couchbase/metricdoc.go

```diff
@@ -0,0 +1,127 @@
+package couchbase
+
+// GAUGE(gauge.bucket.basic.dataUsed): Size of user data within buckets of the specified state that are resident in RAM (%)
```

Comment:
Confirm that these metrics actually do start with `gauge.` and weren't just documented that way.